third-party/sqlalchemy-enum-tables/setup.py

```diff
@@ -11,7 +11,7 @@ def reqs():
 
 
 st.setup(
-	name = "SqlAlchemy Enum Tables",
+	name = "SqlAlchemy-Enum-Tables",
```

Comment:
is this normal python package name formatting? I thought it would be like `sqlalchemy_enum_tables`

Reply:
I can do that.

Reply:
haha ok, but not an emergency

Reply:
Fixed!

Reply:
if i could approve again i would haha